test/wallet-http-test.js

```diff
@@ -883,6 +883,18 @@ describe('Wallet HTTP', function() {
     assert.deepEqual(state, res);
   });
 
+  it('should get all wallet names', async () => {
```

Comment:
https://github.com/handshake-org/hsd/pull/488 will probably get merged soon -- you can wait and make sure but I think these API calls could use the built in client methods!